autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -52,21 +53,23 @@ class BasePipeline(Pipeline):
     __metaclass__ = ABCMeta
 
     def __init__(
-        self,
-        config: Optional[Configuration] = None,
-        steps: Optional[List[Tuple[str, autoPyTorchChoice]]] = None,
-        dataset_properties: Optional[Dict[str, Any]] = None,
-        include: Optional[Dict[str, Any]] = None,
-        exclude: Optional[Dict[str, Any]] = None,
-        random_state: Optional[np.random.RandomState] = None,
-        init_params: Optional[Dict[str, Any]] = None
+            self,
```

Comment:
I think this is over indented -- I do not know why flake did not fail here...

Reply:
I didnt know about it and got no error. I'll fix it